lib/voxpupuli/acceptance/spec_helper_acceptance.rb

```diff
@@ -13,12 +14,21 @@ def configure_beaker(modules: :metadata, &block)
 
   require_relative 'fixtures' if modules == :fixtures
 
-  unless ENV['BEAKER_provision'] == 'no'
+  if ENV['BEAKER_provision'] != 'no'
+    if ENV['BEAKER_provision_puppet'] != 'no'
     block_on hosts, run_in_parallel: true do |host|
       BeakerPuppetHelpers::InstallUtils.install_puppet_release_repo_on(host, collection)
       package_name = BeakerPuppetHelpers::InstallUtils.puppet_package_name(host)
       host.install_package(package_name)
     end
+    else
+      # we assume we provision an instance where puppet-agent ist installed
+      # by default, puppet-agent creates /etc/profile.d/puppet-agent.sh which adds /opt/puppetlabs/bin to PATH
+      # in our non-interactive ssh sessions we manipulate PATH in ~/.ssh/environment, we need to do this step here as well
+      block_on hosts, run_in_parallel: true do |host|
+        host.add_env_var('PATH', '/opt/puppetlabs/bin')
```

Comment:
In my testing this was needed on Debian, but not EL.

Reply:
I think it's fine to always add this. Or do you think I should limit it to Debian here as well?

Reply:
In beaker-puppet I think there was a method `configure_puppet_defaults_on(host)`. Given it was so small I didn't implement it in `beaker_puppet_helpers`, but perhaps it still makes sense.

Overall I wonder how much of this code should live in `voxpupuli-acceptance` and how much in our helper.

In the short term it is probably safe given `add_env_var` can check if it's already present (using grep).